### PR TITLE
[polaris.shopify.com reboot] Feat/add "skip to content" link

### LINKED ISSUE #6069: [polaris.shopify.com reboot] Add a "skip to main content" link
https://css-tricks.com/how-to-create-a-skip-to-content-link/

### PR DESCRIPTION
**Pull request by chazdean**
### WHY are these changes introduced?

Fixes #6069 

### WHAT is this pull request doing?

Screen Shot 2022-06-09 at 3 29 53 PM

Adds a "skip to main content" link that is focusable on tab. When clicked browser will jump to main content of page. 

- Currently only working on Hompage
- Main content on other pages can easily be added by adding `id="main"` to main content container on other pages
- Do we want it on all other pages?

### CODE REVIEW

**Review by gui-santos — APPROVED** [review]
🍨

**Comment by martenbjork** [comment]
Looks great! Maybe add `id="main"` to the `Layout` component? Perhaps on the `<div className={styles.PostContent}>` tag?

File: polaris.shopify.com/src/components/Header/Header.module.scss [hunk 1]
```diff
@@ -76,6 +76,24 @@
   }
 }
 
+.SkipToContentLink {
+  left: 0%;
```
Comment (martenbjork) [inline polaris.shopify.com/src/components/Header/Header.module.scss:80]: Nit: Unit not needed

File: polaris.shopify.com/src/components/Header/Header.module.scss [hunk 2]
```diff
@@ -76,6 +76,24 @@
...
+  background-color: white;
```
Comment (martenbjork) [inline polaris.shopify.com/src/components/Header/Header.module.scss:81]: Use `var(--surface)` to make sure that you get the tokenized color. 

This makes it easy to update the colors across the site in the future. It also enables automatic light / dark mode switching.

File: polaris.shopify.com/src/components/Header/Header.module.scss [hunk 3]
```diff
@@ -76,6 +76,24 @@
...
+  font-weight: var(--font-weight-500);
+  font-size: var(--font-size-100);
+  margin-top: 0.4rem;
+  margin-left: 0.5rem;
+  padding: 1rem;
+  position: absolute;
+  transform: translateY(-100%);
+
+  &:focus {
+    transform: translateY(0%);
```
Comment (martenbjork) [inline polaris.shopify.com/src/components/Header/Header.module.scss:93]: Unit not needed

File: polaris.shopify.com/src/components/Header/Header.module.scss (hunk 1 above)
Comment (chazdean) [reply polaris.shopify.com/src/components/Header/Header.module.scss:80]: @martenbjork I attempted to remove the `left` attribute on the element becomes unaligned 🤔

File: polaris.shopify.com/src/components/Header/Header.module.scss (hunk 2 above)
Comment (chazdean) [reply polaris.shopify.com/src/components/Header/Header.module.scss:81]: 👍🏾

File: polaris.shopify.com/src/components/Header/Header.module.scss (hunk 3 above)
Comment (chazdean) [reply polaris.shopify.com/src/components/Header/Header.module.scss:93]: @martenbjork what would be the alternative for this? With my current understanding this is how the button is hidden and only slides down when focused using tab

**Pull request merged**